exercises/meetup/canonical-data.json

```diff
@@ -1131,7 +1131,7 @@
     },
     {
       "uuid": "00c3ce9f-cf36-4b70-90d8-92b32be6830e",
-      "description": "first Friday of December 2012",
+      "description": "first Friday in a month where it falls on the 7th, the last day of the first week",
```

Comment:
I don't really know why this case is all the way down here. but I don't think moving it should be the job of this PR